www/dart-sass/Portfile

```
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                dart-sass
+version             1.32.8
+categories          www textproc
+platforms           darwin
+license             MIT
+maintainers         @queensferryme openmaintainer
+description         The reference implementation of Sass, written in Dart.
+long_description    ${description}
```

```suggestion
long_description    Dart Sass is the primary implementation of Sass, which means \
                    it gets new features before any other implementation. It's fast, \
                    easy to install, and it compiles to pure JavaScript which makes it \
                    easy to integrate into modern web development workflows.
```

nitpick: Maybe something along these lines from the homepage?

---

fixed, thx for the advice.